src/serde/deserialize_utils.rs

```diff
@@ -62,7 +62,15 @@ pub fn parse_dereference(value: &str) -> Result<ValueAddress, ReferenceParseErro
         1 => parse_dereference_no_offsets(&splitted),
         2 => parse_dereference_with_one_offset(&splitted),
         3 => parse_dereference_with_two_offsets(&splitted),
-        _ => Err(ReferenceParseError::InvalidStringError(String::from(value))),
+        //_ => Err(ReferenceParseError::InvalidStringError(String::from(value))),
```

Comment:
Why is this commented?

Reply:
Resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/279/commits/2f71ce7628a5e9aa9b00ad96d6d5cf0484d895f0